assets/javascripts/discourse/connectors/after-reviewable-flagged-post-body/sift.js.es6

```diff
@@ -0,0 +1,34 @@
+import SiftMod from 'discourse/plugins/discourse-sift/admin/models/sift-mod';
```

Comment:
Are these actions supposed to be wired, or is this old code?

Reply:
No, this is old code that is not used anymore.  Should be able to remove completely